src/module-1/cards-list-v1/index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Card List Design</title>
+  <link rel="stylesheet" href="stylesheets/cards.css">
+  <link rel="stylesheet" href="stylesheets/card.css">
+</head>
+<body>
+  <div class="card-container">
+    <section class="card">
+      <figure>
+        <img class="product" src="images/product.png" alt="Apple Watch">
+      </figure>
+
+      <div class="rate_cost">
+        <div class="rate">
+          <p>3.4</p>
+          <img class="icon" src="images/rate-star.svg" alt="Rate Star">
+        </div>
+
+        <div class="cost">
+          <p>399$</p>
+        </div>
+      </div>
+
+      <article class="description">
+        <h2>Apple Watch Series 4 GPS</h2>
+        <p>Redesigned from scratch and completely revised.</p>
+      </article>
+
+      <div class="action">
+        <button class="wishlist"><img class="icon" src="images/wishlist.svg" alt="Wishlist Heart"> WISHLIST</button>
```

Comment:
minor: you don't need alts for wishlist and cart icons

Reply:
[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)